public/static/docs/tutorials/pipelines.md

```diff
@@ -5,7 +5,7 @@ Let's explore the natural language processing
 ([NLP](https://en.wikipedia.org/wiki/Natural_language_processing)) problem of
 predicting tags for a given StackOverflow question. For example, we want a
 classifier that can predict posts about the Python language by tagging them
-`python`. (This is a short version of the [Tutorial](/doc/tutorials/deep).)
+`python`. (This is a short version of the [Deep Tutorial](/doc/tutorials/deep).)
```

Comment:
should it be the full name then? I think it's fine to put some like of `this tutorial`

Reply:
You mean "Deep Dive Tutorial"? OK, sure, good catch. Done in af6fc639

We're already in a tutorial so maybe saying "this tutorial" to link to another tutorial is strange?